### DISCUSSION TITLE
Finish Font

### DISCUSSION THREAD

**Discussion by cpyarger** [malcolmholmes/pico-clock-green-python#8] [Ideas]
It would be really nice to have at least all of the letters available. 

Preferably both upper and lower case

**Comment by malcolmholmes** [comment]
Fancy taking a stab at it? The set of letters present at the moment are just those pulled in from the C code (with minor adjustments)

**Comment by cpyarger** [comment]
Sure. I can at least get the upper case ones going I think. It doesn't look
difficult just tedious

**Comment by cpyarger** [comment]
I issued pr #14  to finish adding upper case letters. the Q is a bit weird but its what i could come up with

**Comment by GeoffRiley** [comment]
I could throw a table together for the lowercase characters in a 7x5 grid: I've done it before on other systems in C and Pascal so I'll have the data tables in my archive somewhere.  Based on the old TRS-80 and Epson FX fonts! Of course it's impossible to have descenders, so any chars with such features will 'float' up in the character square.  Give me a nod if you'd like me to go for it.

**Comment by cpyarger** [comment]
Sounds good to me. It would be a great help. And if you have any
suggestions for the upper case characters I'd love to hear them

**Comment by GeoffRiley** [comment]
Sure thing, I'll dig the table out: it has both upper and lower case so I'll do a plot of the whole set and you can choose if you want to take the upper case and numerals too.  Most of the glyphs that I've seen are very similar already anyway.  It might take me a little while to find the tables in my archive—long before git, I was using plain 'rcs' back then!
